content/tools.md

```diff
@@ -0,0 +1,36 @@
+# Tools
```

Comment:
I think it is supposed to have a blurb like this:
```
---
Title:       Tools
Author:      Sven Trittler
Date:        2024-05-28
Template:    plain_markdown
---
```
on top, but it does work despite this not being there 🤔

(Note that I just guessed at the contents apart from the `Template: plain_markdown ` line)